### DISCUSSION TITLE
Using filebrowser command line inside a pod

### DISCUSSION THREAD

**Discussion by joebar72** [filebrowser/filebrowser#2273] [Q&A]
Hi, I am trying to run filebrowser inside a pod. Via command line I am not able to do anything. Every command gives me a timeout including export ones. Anyway I do not understand how can the filebrowser configuration works after then I stopped it. Via command line seams that only "version" is working fine. Any idea?

This is an example:

/opt $ /filebrowser users --help
Users management utility.

Usage:
filebrowser users [command]

Available Commands:
add Create a new user
export Export all users to a file.
find Find a user by username or id
import Import users from a file
ls List all users.
rm Delete a user by username or id
update Updates an existing user

Flags:
-h, --help help for users

Global Flags:
-c, --config string config file path
-d, --database string database path (default "./filebrowser.db")

Use "filebrowser users [command] --help" for more information about a command.
/opt $ /filebrowser users ls
2022/12/28 15:41:30 timeout
/opt $ /filebrowser version
File Browser v2.23.0/02db83c7
/opt $

**Comment by akupiec** [comment]
Responding to further generations :)

I stumbled upon the same problem while running it with Docker Compose. I assumed it was due to a busy database. Here's a workaround:

1. Stop the currently running container.
2. Run the necessary commands before FileBrowser starts up. For example:
```
docker run --rm \
    -v filebrowser_database:/database \
    -v filebrowser_config:/config \
    filebrowser/filebrowser \
    config --help
```

of course you need to attach to proper database & config volumes.
